docker-compose.dev-refresh.yml

```diff
@@ -0,0 +1,51 @@
+version: "3.4"
+
+services:
+  # dev-refresh service will start up a grant server bound to host port 3334
+  # which allows one to do `docker restart grant-dev-refresh` when the user
+  # wants to "restart" the service running new code.  This is especially helpful
+  # when you hook it up to `fswatch` type utilities, causing a re-run of `go run`
+  # every time a file changes.
+  dev-refresh:
+    container_name: grant-dev-refresh
+    image: golang:1.13.4
+    ports:
+      - "3334:3333"
```

Comment:
One problem that I have is that the server that runs in `go run` is on port `3333`. I map mine to run on the same port `3333:3333`

Reply:
I made it :3334 because we have another service in docker-compose that binds to 3333 (grant-web)

Reply:
So does this work locally for you outside the docker container? Because of the the following lines in `grant-server/main.go` I can't access the web server from my machine.

https://github.com/brave-intl/bat-go/blob/2ef89055c431a6155d56dc8d1c0349d5d64af09c/bin/grant-server/main.go#L224-L225

Reply:
@corymcdonald docker should be forwarding from localhost:3334 to grant-dev-refresh
also @husobee i believe reputation server is already on there. if you wouldn't mind going one higher that would be swell

Reply:
:+1: will do in next commit.